delta-app/src/main/java/io/cdap/delta/store/HCFSStateStore.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.delta.store;
+
+import com.google.common.io.ByteStreams;
+import io.cdap.delta.api.DeltaPipelineId;
+import io.cdap.delta.api.Offset;
+import io.cdap.delta.app.DeltaWorkerId;
+import io.cdap.delta.app.OffsetAndSequence;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+/**
+ * Stores replicator state based on hadoop filesystem .
+ */
+public class HCFSStateStore implements StateStore {
```

Comment:
is this just the previous StateStore implementation except renamed? If not what are the differences?

Reply:
It's just the rename. No other difference